blog/models.py

```diff
@@ -1,3 +1,20 @@
 from django.db import models
 
-# Create your models here.
+class ArticleCategory(models.Model):
+    
+    name = models.CharField(max_length=255)
+    description = models.TextField()
+
+    def __str__(self):
+        return f'{self.name}'
+
+class Article(models.Model):
+
+    title = models.CharField(max_length=255)
+    category = models.ForeignKey(ArticleCategory, on_delete=models.SET_NULL)
+    entry = models.TextField()
+    created_on = models.DateTimeField()
+    updated_on = models.DateTimeField()
```

Comment:
use `auto_now` to automatically set `updated_on` to whenever an instance of `Article` is updated.